client/asset/bch/spv.go

```diff
@@ -29,6 +29,9 @@ import (
 	"github.com/btcsuite/btcwallet/waddrmgr"
 	btcwallet "github.com/btcsuite/btcwallet/wallet"
 	"github.com/btcsuite/btcwallet/wtxmgr"
+	neutrino "github.com/dcrlabs/neutrino-bch"
+	labschain "github.com/dcrlabs/neutrino-bch/chain"
```

Comment:
Is the bchwallet/chain still needed?  It's only used at `chain.UseLogger` I think.  Does it still get used indirectly?

Reply:
Looks like there's at least one place left where that package logger could potentially be used. In `(*BlockFilterer).FilterTx`